packages/integration-tests/src/test-project-setup/setup_dir_as_esm_module.ts

```diff
@@ -29,4 +29,20 @@ export const setupDirAsEsmModule = async (absoluteDirPath: string) => {
     stdio: 'inherit',
     cwd: absoluteDirPath,
   });
+
+  const pathsObj = {
+    // The path here is coupled with backend-function's generated typedef file path
+    '@env/*': ['../.amplify/function-env/*'],
+  };
+
+  const tsConfigPath = path.resolve(absoluteDirPath, 'tsconfig.json');
+  const tsConfigContent = (await fs.readFile(tsConfigPath, 'utf-8')).replace(
+    /\/\*[\s\S]*?\*\/|([^:]|^)\/\/.*$/gm,
```

Comment:
can you add some comment here for what this is suppose to match?

Reply:
I have a comment a couple of lines down that this will remove all comments. I'll move it up to line 40 for better readability